frontend/packages/helm-plugin/console-extensions.json

```diff
@@ -12,12 +12,31 @@
       "abbr": "HCR"
     }
   },
+  {
+    "type": "console.model-metadata",
+    "properties": {
+      "model": {
+        "group": "helm.openshift.io",
+        "version": "v1beta1",
+        "kind": "ProjectHelmChartRepository"
+      },
+      "label": "%helm-plugin~Project Helm Chart Repository%",
+      "labelPlural": "%helm-plugin~Project Helm Chart Repositories%",
+      "abbr": "PHCR"
+    }
+  },
   {
     "type": "console.flag",
     "properties": {
       "handler": { "$codeRef": "helmUtils.detectHelmChartRepositories" }
     }
   },
+  {
+    "type": "console.flag/namespaced",
+    "properties": {
+      "handler": { "$codeRef": "helmUtils.useProjectHelmChartRepositories" }
```

Comment:
Can we not have one handler that fetches both the cluster scoped and namespace scoped repos and sets the flag(FLAG_OPENSHIFT_HELM) if there is(are) chart repo(s) available? cc @rohitkrai03